### PR TITLE
feat: actually load the script from export config

### PR DESCRIPTION
**Pull request by odinuv**
third step of https://github.com/keboola/data-loader/issues/26

actually i was overly defensive, i could've done it in the first step without problem

### CODE REVIEW

File: src/ConfigValidator.php [hunk 1]
```diff
@@ -111,6 +111,7 @@ private function validateExportConfig(): void
         }
         $this->input = $configData['storage']['input'];
         $this->type = $configData['parameters']['type'];
+        $this->script = implode("\n", $configData['parameters']['script']);
```
Comment (pivnicek) [inline src/ConfigValidator.php:114]: do we want to validate that `['parameters']['script']` is an array?

**Review by pivnicek — APPROVED** [review]
I think it's fine, just thought about maybe some malformed configuration

File: src/ConfigValidator.php (hunk 1 above)
Comment (odinuv) [reply src/ConfigValidator.php:114]: yes, we do! 

https://github.com/keboola/data-loader/blob/master/src/ExportConfig.php#L30

Comment (pivnicek) [reply src/ConfigValidator.php:114]: LOL, sorry 🤦‍♂️ 👍

**Pull request merged**